ports/openexr/portfile.cmake

```
@@ -5,27 +5,25 @@ endif()
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO openexr/openexr
-  REF ed64d5467dee9763f28baf300f7699e6288b9f5f
-  SHA512 549d37ed1ef4d1ff7e732d583f7213ee15c7f92625aea9fd65345e4c5b854902c02e5940d0692b1af5ae0a02abf46aaefea2662db2389d1b2fb4264a373baac2
+  REF 918b8f543e81b5a1e1aca494ab7352ca280afc9e
```

```suggestion
  REF 420da6a57a3ecaeab481b993dd590eeec8fe0c52 # v3.1.5
```
We should update to the latest version.